api/redhatopenshift/HcpCluster.Management/hcpCluster-models.tsp

```diff
@@ -456,7 +459,35 @@ model NodePoolAutoScaling {
   @minValue(0)
   max?: int32;
 }
+/** Disk encryption settings for the OS and data disks */
+model Encryption {
+  /** Infrastructure-level encryption for the disks. Defaults to "disabled". */
+  infrastructureEncryption?: "enabled" | "disabled" | string = "disabled";
```

Comment:
Unrelated to the syntax but rather to the enum values themselves, can you please capitalize them?
I just fixed this up across the API in another PR: https://github.com/Azure/ARO-HCP/pull/1965
```suggestion
  infrastructureEncryption?: "Enabled" | "Disabled" | string = "Disabled";
```

Reply:
Done